test/e2e/network/service.go

```diff
@@ -2706,6 +2692,40 @@ var _ = common.SIGDescribe("Services", func() {
 		}
 	})
 
+	ginkgo.It("should support externalTrafficPolicy=Local for type=NodePort", func(ctx context.Context) {
```

Comment:
this is failing, we have this test here https://github.com/kubernetes/kubernetes/pull/123622 where we removed one of the use cases